packages/replay/src/bin.ts

```diff
@@ -71,42 +82,27 @@ program
   )
   .action(commandUploadAllRecordings);
 
-program
-  .command("view <id>")
+commandWithGlobalOptions("view <id>")
   .description("Load the devtools on a recording, uploading it if needed.")
-  .option("--directory <dir>", "Alternate recording directory.")
-  .option("--server <address>", "Alternate server to upload recordings to.")
   .option("--api-key <key>", "Authentication API Key")
   .action(commandViewRecording);
 
-program
-  .command("view-latest")
+commandWithGlobalOptions("view-latest")
   .description("Load the devtools on the latest recording, uploading it if needed.")
-  .option("--directory <dir>", "Alternate recording directory.")
-  .option("--server <address>", "Alternate server to upload recordings to.")
   .option("--api-key <key>", "Authentication API Key")
   .action(commandViewLatestRecording);
 
-program
-  .command("rm <id>")
+commandWithGlobalOptions("rm <id>")
   .description("Remove a specific recording.")
-  .option("--directory <dir>", "Alternate recording directory.")
   .action(commandRemoveRecording);
 
-program
-  .command("rm-all")
-  .description("Remove all recordings.")
-  .option("--directory <dir>", "Alternate recording directory.")
-  .action(commandRemoveAllRecordings);
+program.command("rm-all").description("Remove all recordings.").action(commandRemoveAllRecordings);
```

Comment:
Shouldn't this also use `commandWithGlobalOptions()`?

Reply:
Yep! Missed it

Reply:
Fixed!